src/payment/strategies/opy/opy-library.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * https://widgets.openpay.com.au/config
+ */
+export interface OpyWidgetConfig {
+    /**
+     * The region your store is located in.
+     */
+    region: 'AU' | 'UK' | 'US';
+    /**
+     * The currency symbol used in the widgets.
+     */
+    currency: '$' | '£';
+    /**
+     * An array of available plan tiers you have available in months.
+     */
+    planTiers: number[];
+    /**
+     * The minimum eligible amount required before Opy is eligible.
+     */
+    minEligibleAmount: number;
+    /**
+     * The maximum eligible amount required before Opy is eligible.
+     */
+    maxEligibleAmount: number;
+    /**
+     * Type of your store.
+     */
+    type: 'Online';
+}
+
+export interface OpyWidgets {
```

Comment:
Since this is a singular value, do you think it makes sense to call it `OpyWidget`